daily-patch-information/src/main/java/org/wso2/engineering/efficiency/patch/analysis/EmailsSender.java

```diff
@@ -0,0 +1,161 @@
+        String emailSubject;
+        String emailHeaderHTML;
+        if (isMailOnCustomerReportedIssues) {
+            urlToJIRAFilter = configuration.getUrlToJIRAFilterCustomer();
+            emailSubject = SUBJECT_CUSTOMER;
+            emailHeaderHTML = MAIN_HEADER_CUSTOMER;
+        } else {
+            urlToJIRAFilter = configuration.getUrlToJIRAFilterInternal();
+            emailSubject = SUBJECT_INTERNAL;
+            emailHeaderHTML = MAIN_HEADER_INTERNAL;
+        }
+
+        ArrayList<JIRAIssue> jiraIssues = accessingJIRA(urlToJIRAFilter, configuration.getJiraAuthentication());
+        accessingPMT(configuration.getPmtConnection(), configuration.getDbUser(), configuration.getDbPassword(),
+                jiraIssues);
+        emailHeaderHTML += jiraIssues.size() + MAIN_HEADER_END;
+        String emailBodyHTML = EmailContentCreator.getEmailContentCreator().getEmailBody(jiraIssues, emailHeaderHTML);
+        //send mail
+        try {
+            EmailSender.getEmailSender().sendMessage(emailBodyHTML, emailSubject, configuration.getEmailUser(),
+                    configuration.getToList(), configuration.getCcList());
+            LOGGER.info("Successfully sent email with patch information.");
+        } catch (PatchInformationException e) {
+            String errorMessage = "Failed to send email.";
+            LOGGER.error(errorMessage, e);
+            throw new PatchInformationException(errorMessage, e);
+        }
+    }
+
+    /**
+     * Gets the JIRA issues returned by the JIRA filter.
+     *
+     * @param urlToJIRAFilter    url to the results from applying the JIRA filter.
+     * @param jiraAuthentication Basic authentication.
+     * @return JIRA issues.
+     * @throws PatchInformationException could not extract JIRA issues.
+     */
+    private ArrayList<JIRAIssue> accessingJIRA(String urlToJIRAFilter, String jiraAuthentication)
+            throws PatchInformationException {
+
+        try {
+            ArrayList<JIRAIssue> jiraIssues;
+            jiraIssues = new ArrayList<>(JIRAAccessor.getJiraAccessor().getIssues(urlToJIRAFilter, jiraAuthentication));
+            LOGGER.info("Successfully extracted JIRA issue information from JIRA.");
+            return jiraIssues;
+
+        } catch (PatchInformationException e) {
+            String errorMessage = "Failed to extract JIRA issues from JIRA.";
+            LOGGER.error(errorMessage, e);
+            throw new PatchInformationException(errorMessage, e);
+        }
+    }
+
+    /**
+     * Assigns patches to the JIRA issues.
+     *
+     * @param pmtConnection   pmt connection.
+     * @param pmtUserName     pmt username.
+     * @param pmtUserPassword pmt password.
+     * @param jiraIssues      JIRA issues returned by the JIRA filter.
+     * @throws PatchInformationException Could not extract Patch information from pmt.
+     */
+    private void accessingPMT(String pmtConnection, String pmtUserName, String pmtUserPassword,
```

Comment:
As mentioned above, it should be accessPMT. Please don't use 'ing' for method names, it should be simple verbs.